lib/install/data/data.go

```diff
@@ -373,6 +373,8 @@ func (d *Data) CopyNonEmpty(src *Data) error {
 
 	d.RegistryCAs = src.RegistryCAs
 
+	d.SyslogConfig = src.SyslogConfig
```

Comment:
If --syslog-address is not set, will it be reset to empty during configure?

Reply:
In `func (*Configure) copyChangedConf ()`, I added code `if c.SyslogConfig.Addr != nil` to decide whether to change extraconfig.   
This is the same way as RegistryCA does.